Packs/CortexResponseAndRemediation/Playbooks/silent-playbook-A_user_executed_multiple_LDAP_enumeration_queries_Test.yml

```diff
@@ -660,8 +660,7 @@ tasks:
     skipunavailable: false
     task:
       brand: ''
-      description: Checks if the user's creation date in AD happened 24 hours or less
-        since the time that this alert occurred.
+      description: Checks if the user's creation date in AD happened 24 hours or less since the time that this alert occurred.
```

Comment:
Checks whether the user was created in Active Directory within 24 hours before the alert occurred.